src/v2/Apps/Home/homeRoutes.tsx

```diff
@@ -10,7 +10,7 @@ const HomeApp = loadable(
 export const homeRoutes: AppRouteConfig[] = [
   {
     theme: "v3",
-    path: "/home",
+    path: "//",
```

Comment:
This is the only slightly weird thing. I guess maybe this warrants a comment. @damassi do you know why this has to happen? We're stripping the leading `/`?

Reply:
Not sure if you saw the slack thread, but reposting here: https://github.com/artsy/force/blob/master/src/v2/System/Router/buildAppRoutes.tsx#L71-L75. 

Perhaps this has something to do with it?

Reply:
> If the last character is a `/` trim the first character of the path.

What does this even mean? Do you happen to know the why of this?

Reply:
These are the routes that get mounted:

```
/artist/:artistID
/artist-series/:slug
/artists
/artwork/:artworkID/:optional?
/auctions
/buyer-guarantee
/collect/:medium?
/collect/color/:color?
/collections
/collection/:slug
/consign
/consign/offer/:offerID
/user/conversations
/user/conversations/:conversationID
/example
/fair/:slug
/fair/:slug
/fairs
/art-fairs
/feature/:slug
/gene/:slug
/
/identity-verification/processing
/identity-verification/error
/identity-verification/:id
/order(2|s)/:orderID
/partner/:partnerId
/user/payments
/user/purchases
/search
/show/:slug
/show/:slug
/shows2
/tag/:slug
/unsubscribe
/viewing-rooms
/viewing-room/:slug
/debug
```

The only one that would actually match this is the new homepage. I think this code is safe to remove?

Reply:
eea0223c9562c6a538219be6f4c7fd10e00300e1

Reply:
I'd be triple sure its safe to remove, but 👍

Reply:
Lemme try to hunt down the PR it was introduced in.

Reply:
This was added in this commit: https://github.com/artsy/reaction/commit/5ee5f1fee212439b6ec8bd6e02a4fbdf93ddf044 / this PR: https://github.com/artsy/reaction/pull/3043

I don't know what its original intent was; but this code hasn't executed in a very long time.